Tests/CaseIterableTests.swift

```diff
@@ -0,0 +1,166 @@
+import XCTest
+import SweetUIKit
+
+class CaseIterableTests: XCTestCase {
+
+    // MARK: - Test Enums
+
+    enum ZeroOptions: CaseIterable {}
+
+    enum OneOption: CaseIterable {
+        case first
```

Comment:
I find it a little confusing since `.first` is also a method on an array. Maybe name it `.firtstCase` `.secondCase` etc? Bit less clean but at least non-confusing?

Reply:
Good point. Will rename.